Remove failing overlay test

dmcgowan opened the pull request:
Diff apply is sometimes producing a different change list causing the tests to fail.
Overlay has a known issue calculating diffs of files which occur within the same second they were created.

See #21555 for information on related issue

tiborvass commented:
LGTM

tonistiigi commented:
LGTM

tonistiigi commented:
@dmcgowan  `TestOverlayChanges` is failing as well in https://jenkins.dockerproject.org/job/Docker%20Master%20(arm)/2064/console

dmcgowan commented:
@tonistiigi: thanks, I didn't see that one, I will add that to this PR

tiborvass commented:
@dmcgowan why not use t.Skipf ?

dmcgowan committed:
Remove failing overlay test

Diff apply is sometimes producing a different change list causing the tests to fail.
Overlay has a known issue calculating diffs of files which occur within the same second they were created.

Signed-off-by: Derek McGowan <derek@mcgstyle.net> (github: dmcgowan)

dmcgowan commented:
@tiborvass: I would prefer to use skip, I'll update

thaJeztah commented:
Experimental timed out after all tests completed, so merging
